### PR TITLE
quit one drug and add more diagnosis for mental health

### PR DESCRIPTION
**Pull request by Joseph-Developer**

### CODE REVIEW

File: configuration/conceptsets/.~lock.diagnoses-mental-health-set.csv# [hunk 1]
```diff
@@ -0,0 +1 @@
+,joseph,developer,16.09.2022 20:33,file:///home/joseph/.config/libreoffice/4;
```
Comment (mogoodrich) [inline configuration/conceptsets/.~lock.diagnoses-mental-health-set.csv#:1]: Probably committed by accident... :)

File: configuration/concepts/.~lock.diagnoses-mental-health.csv# [hunk 2]
```diff
@@ -0,0 +1 @@
+,joseph,developer,16.09.2022 20:32,file:///home/joseph/.config/libreoffice/4;
```
Comment (mogoodrich) [inline configuration/concepts/.~lock.diagnoses-mental-health.csv#:1]: Another file committed by accident? :)

**Review by mogoodrich — COMMENTED** [review]
LGTM, @Joseph-Developer ... just a couple of files likely committed by accident!

As a side note, I forget if I pointed this out to you, but there's a issue with the way Iniz works that you should be aware of:

https://github.com/mekomsolutions/openmrs-module-initializer/issues/198

(it shouldn't be a problem in this case since both the mental health concept and concept set files were updatd, but just as an fyi)

File: configuration/concepts/.~lock.diagnoses-mental-health.csv# (hunk 2 above)
Comment (Joseph-Developer) [reply configuration/concepts/.~lock.diagnoses-mental-health.csv#:1]: https://github.com/PIH/openmrs-config-ces/commit/98be643e110b39335f776562b67d5d9bc29bdf2a,

yes, I did not realize an apology

Comment (mogoodrich) [reply configuration/concepts/.~lock.diagnoses-mental-health.csv#:1]: No worries! :)

**Pull request merged**